tests/js/spec/views/eventsV2/utils.spec.jsx

```diff
@@ -19,23 +19,23 @@ describe('eventTagSearchUrl()', function() {
   it('adds a query', function() {
     expect(getEventTagSearchUrl('browser', 'firefox', location)).toEqual({
       pathname: location.pathname,
-      query: {query: 'browser:"firefox"'},
+      query: {query: 'browser:firefox'},
```

Comment:
@markstory Was this unit test incorrect before?

Reply:
No, I changed the behavior. Previously the event tags table would always quote, but the quotes are only required if the value has a space in it.